crates/rolldown_utils/src/base64.rs

```diff
@@ -3,3 +3,7 @@ use base64::{engine::general_purpose, Engine as _};
 pub fn to_url_safe_base64(input: impl AsRef<[u8]>) -> String {
   general_purpose::URL_SAFE_NO_PAD.encode(input)
 }
+
+pub fn to_standard_base64(input: impl AsRef<[u8]>) -> String {
+  general_purpose::STANDARD.encode(input)
```

Comment:
Could you verify that we are using the same standard as esbuild?

Reply:
Yes, the [base64 encoding method](https://github.com/evanw/esbuild/blob/67cbf87a4909d87a902ca8c3b69ab5330defab0a/internal/bundler/bundler.go#L284) of base64 loader in esbuild is the standard method in [RFC 4648](https://www.rfc-editor.org/rfc/rfc4648.html), and we are using the same standard according to the [doc](https://docs.rs/base64/latest/base64/#encoding-alphabet) of `base64` crate